requirements/dev.txt

```diff
@@ -7,7 +7,6 @@
 argparse==1.4.0           # via caniusepython3
 astroid==1.6.6            # via pylint, pylint-celery
 backports.functools-lru-cache==1.6.1  # via astroid, caniusepython3, isort, pylint
-backports.os==0.1.1       # via path.py
```

Comment:
It looks like you ran `make upgrade` under Python 3.5, I think these are still needed under Python 2.

Reply:
Still need to get back these 2 Python-2-only dependency pins.

Reply:
I ran `make upgrade` in python2 but some how its removing automatically.